fragments/platform/rhel8/section-data/server-dvd-link.ks

```diff
@@ -0,0 +1,7 @@
+# Create a variable in the pre/post section with link to a Fedora rawhide server DVD.
```

Comment:
Fedora rawhide -> RHEL, or maybe just with "link to a server DVD"

Reply:
Fixed.